style.css

```diff
@@ -0,0 +1,698 @@
+  margin-bottom: 50px;
+  opacity: 0.7;
+  z-index: -1;
+}
+
+.menu-bar {
+  display: none;
+}
+
+#container {
+  display: flex;
+  flex-direction: column;
+  background-image: url("images+/fondopantalla1.jpg");
+}
+
+#different-parts {
+  color: antiquewhite;
+  font-size: 30px;
+  text-align: center;
+  margin-bottom: 60px;
+}
+
+.living-room {
+  width: 358px;
+  height: 70px;
+  background-color: rgb(95, 106, 107);
+  color: rgb(245, 77, 48);
+  font-size: 24px;
+  display: flex;
+  flex-direction: row;
+}
+
+.title-room {
+  margin-top: 10px;
+}
+
+#text-living {
+  font-size: 11px;
+  width: 220px;
+  padding-left: 42px;
+  color: white;
+}
+
+#text-bedroom {
+  font-size: 11px;
+  width: 280px;
+  padding-left: 58px;
+  color: white;
+  margin-bottom: 10px;
+}
+
+#text-kitchen {
+  font-size: 11px;
+  width: 280px;
+  padding-left: 23px;
+  color: white;
+}
+
+#text-toilette {
+  font-size: 11px;
+  width: 280px;
+  padding-left: 23px;
+  color: white;
+}
+
+.living-room:hover {
+  background: #ee3d3d;
+  color: white;
+  border: 1px solid #fdfeff;
+}
+
+.living-room:active {
+  background: #6070ff;
+  color: white;
+  border: 1px solid #cc3f3f;
+}
+
+.line {
+  size: 3px rgb(245, 77, 48);
+  width: 50px;
+}
+
+#title-speakers {
+  margin-top: 90px;
+  margin-left: 40px;
+  font-size: 25px;
+  margin-bottom: 80px;
+}
+
+.photos-decoration {
+  display: flex;
+  flex-direction: row;
+  list-style-type: none;
+  margin: 12px;
+  padding: 10px;
+  margin-top: 45px;
+}
+
+.images-photo-decoration {
+  margin-left: 10px;
+}
+
+#image-project {
+  width: 180px;
+  height: 180px;
+}
+
+#image-project1 {
+  width: 180px;
+  height: 180px;
+}
+
+#image-project2 {
+  width: 180px;
+  height: 180px;
+}
+
+#image-project3 {
+  width: 180px;
+  height: 180px;
+}
+
+#content-members {
+  display: flex;
+  flex-direction: row;
+  margin-left: -30px;
+}
+
+.membersDavid {
+  display: flex;
+  flex-direction: row;
+}
+
+.members {
+  margin-top: 15px;
+}
+
+#title-main-speakers {
+  margin-left: 15px;
+}
+
+#subtitle-main-speakers {
+  margin-left: 15px;
+  color: rgb(245, 77, 48);
+}
+
+#text-speakers {
+  margin-left: 5px;
+  margin-bottom: 55px;
+}
+
+#title-main-speakers1 {
+  margin-left: 15px;
+}
+
+#subtitle-main-speakers1 {
+  margin-left: 15px;
+  color: rgb(245, 77, 48);
+}
+
+#text-speakers1 {
+  margin-left: 5px;
+  margin-bottom: 55px;
+}
+
+#title-main-speakers2 {
+  margin-left: 15px;
+}
+
+#subtitle-main-speakers2 {
+  margin-left: 15px;
+  color: rgb(245, 77, 48);
+}
+
+#text-speakers2 {
+  margin-left: 5px;
+  margin-bottom: 55px;
+}
+
+#title-main-speakers3 {
+  margin-left: 15px;
+}
+
+#subtitle-main-speakers3 {
+  margin-left: 15px;
+  color: rgb(245, 77, 48);
+}
+
+#text-speakers3 {
+  margin-left: 5px;
+  margin-bottom: 55px;
+}
+
+.line2 {
+  size: 3px rgb(245, 77, 48);
+  width: 50px;
+  margin-left: 15px;
+}
+
+#container-partners {
+  background-color: rgb(86, 90, 90);
+  margin-bottom: 20px;
+  margin-top: 10px;
+}
```

Comment:
- [ ] Great job working on the requested changes 👏 . Unfortunately, one was missed. As the previous reviewer mentioned kindly remove the `partner section` in the `mobile version` `home page` 🙏 

![image](https://user-images.githubusercontent.com/74814780/191872123-95432146-ae94-41af-be14-127ad6add1f2.png)